### ISSUE TITLE
Add events extra category/value as dimensions for event counts view

### ISSUE THREAD

**Issue by wlach** [mozilla/lookml-generator#319]
It would be sometimes be handy to get counts of category/values in the event counts view, for example to produce visualizations like this one:

https://sql.telemetry.mozilla.org/queries/83002/source#205684

It appears that we already have a "view" on this property in Looker, probably just a matter of wiring it up. One complication or limitation is that there could be potentially be up to 20 "extras" per an event. As with the funnel analysis explore, only supporting one seems like a reasonable compromise:

https://github.com/mozilla/looker-hub/blob/70fd0c0b99e5bbd1ba5c2c8f0d6f8f20ca6897ba/firefox_ios/views/events_table.view.lkml#L521

**Comment by wlach** [comment]
It appears this is already implemented for the desktop (non-Glean) version of this view.

**Comment by vesta0** [comment]
Is this work on the roadmap for H1?

**Comment by scholtzan** [comment]
> Is this work on the roadmap for H1?

Not at the moment, but if we actually have a need for this, then priority could be bumped up. Is this something you'd want @vesta0?

**Comment by vesta0** [comment]
Yes, without this capability, mobile is not able to monitor and track a large percentage of Android usage metrics.

**Comment by scholtzan** [comment]
I'll see to it that this gets triaged during our next Looker workgroup meeting on Thursday.

**Comment by scholtzan** [comment]
@vesta0 This should now be available. For example see https://mozilla.cloud.looker.com/x/1hWTUZDliwuw40obSfLFvZ which accesses the event extra values
